### PR TITLE
Feature/text locator result source indicator

### PR DESCRIPTION
**Pull request by warm-coolguy**
## Summary

The results in the TextLocator client may seem confusing at times, since not all places to each piece of literature, nor all pieces of literature to each place are shown at all times. However, this must stay this way; else, we'd pretty much load the whole database due to overlap chains. (Literature has place, which has other literature, which has other places, which ...)

Still, users may have problems in understanding this in detail. To help with this, a minimal indicator has been placed in the UI so that the source for the currently visible results is kept. This may later be extended by a more sophisticated description in e.g. a tooltip, depending on the customer review. For now, it's mostly meant to transport the idea and implement the feature roughly.

## Instructions for local reproduction and review

Run TextLocator and try all different ways to display the search results. All major paths (search for literature/toponym/geometry) have their own respective display string.

## Relevant tickets, issues, et cetera

Based/Blocks on https://github.com/Dataport/polar/pull/172

### CODE REVIEW

**Review by dopenguin — COMMENTED** [review]
🏓 @warm-coolguy 

A changelog entry is missing.

**Comment by warm-coolguy** [comment]
> 🏓 @warm-coolguy
> 
> A changelog entry is missing.

🏓 The changelog entry has been unmissingfied.

File: packages/clients/textLocator/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Breaking: Adapt client to new backend API. Previous versions are no longer runnable due the backend API change.
 - Feature: Implement document search, that is, the AddressSearch bar now also offers results for documents and clicking on one will retrieve a list of toponyms from the backend and resolve them against the gazetteer. Previously, we had a "get all documents regarding place" functionality. Now, a "get all places regarding document" feature is implemented. Additionally to the AddressSearch, this can be triggered on documents found in the previously implemented way; the "get all places regarding document" functionality is available from within the GeoSearch result display.
+- Feature: The result display now features a display regarding what kind of search interaction the current search results stem from. This is meant to enhance interpretability of the results' scope.
```
Comment (dopenguin) [inline packages/clients/textLocator/CHANGELOG.md:7]:
```suggestion
- Feature: Add a display regarding what kind of search interaction the current search results stem from to the result display. This is meant to enhance interpretability of the results' scope.
```
🎩

**Review by dopenguin — APPROVED** [review]
🚀 @warm-coolguy 

Also added a suggestion as I prefer imperative in changelogs

**Pull request merged**